posthog/tasks/test/test_update_cache.py

```diff
@@ -63,3 +67,59 @@ def test_refresh_dashboard_cache(self, patch_update_cache_item: MagicMock, patch
         self.assertIsNotNone(DashboardItem.objects.get(pk=item_do_not_cache.pk).last_refresh)
         self.assertEqual(cache.get(item_key)["result"][0]["count"], 0)
         self.assertEqual(cache.get(funnel_key)["result"][0]["count"], 0)
+
+    def _test_refresh_dashboard_cache_types(
```

Comment:
ensures that the retention/trend/stickiness filters trigger a proper dashboarditem refresh